Procfile

```diff
@@ -1 +0,0 @@
-web: npm start
```

Comment:
This is only needed when publishing to Heroku... but we're not doing that so this is unnecessary.